cluster/eks.go

```diff
@@ -1215,8 +1215,20 @@ func (c *EKSCluster) GetConfigSecretId() string {
 }
 
 func (c *EKSCluster) GetK8sIpv4Cidrs() (*pkgCluster.Ipv4Cidrs, error) {
-	//TODO
-	return nil, errors.New("not implemented")
+	// On EKS the services IP range is chosen from the two possible ranges below
+	// source: https://forums.aws.amazon.com/thread.jspa?messageID=859958
+	eksServiceClusterIPRanges := []string{"172.20.0.0/16", "10.100.0.0/16"}
+
+	// On EKS the pods IP range is coming from the CIDR's of the subnets
+	var eksPodIPRanges []string
+	for _, subnet := range c.GetModel().EKS.Subnets {
```

Comment:
If the cluster is created with default settings than the default CIDR defined in the CloudFormation template is used. In that case the subnet CIDR is not persisted to the database. Currently we persist to db only the CIDR passed in by user.